students/Mintae_K/lesson04/mailroom2.py

```diff
@@ -0,0 +1,69 @@
+# Sending a Thank you
+def sendingthank(donors):
+    f_name = input('Please type the Full Name or list> ')
+    while f_name == 'list':
+        print(' ')
+        for x in donors:
+            print(x)
+        print(' ')
+        f_name = input('Please type the Full Name or list> ')
+    if donors.get(f_name, False):
+        s = 'Existing donor, please type the donation amount from {}> '
+        amount = input(s.format(f_name))
```

Comment:
you can move this line and line 16 to above line 10; they are duplicated

Reply:
the prompts (my text to display for user) are different for line 12 and line 16. Additionally, if statement is dictating which prompt to show. Thus cannot just simply move them outside of if statement.